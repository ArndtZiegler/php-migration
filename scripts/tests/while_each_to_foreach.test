# Standard cases
while(list($key, $value) = each ( $attrs )) {
while (list( $key, $val )=each($_POST))
while(list($file,$info)=each($this->images))

# Missing key
while( list(, $line) = each($lines) ) {
while( list(, $line ) = each( $lines ) ) {

while (list(,$aro_group_id) = @each($aro_group_ids)) {