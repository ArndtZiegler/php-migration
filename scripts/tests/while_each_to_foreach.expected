# Standard cases
foreach( $attrs as $key => $value) {
foreach ($_POST as $key => $val )
foreach($this->images as $file => $info)

# Missing key
foreach($lines as $line) {
foreach( $lines as $line ) {

foreach ($aro_group_ids as $aro_group_id) {